xcodeproj/internal/target_properties.bzl

```
@@ -102,6 +102,19 @@ def process_modulemaps(*, swift_info):
         files = uniq(modulemap_files),
     )
 
+def process_codesignopts(*, codesignopts, build_settings):
+    """ Logic for processing code signing options
+
+    Args:
+        codesignopts: A `list` of code sign options
+        build_settings: build settings of the target
+
+    Return:
+        The modified build settings object
+    """
+    if codesignopts and build_settings != None:
+        set_if_true(build_settings, "OTHER_CODE_SIGN_FLAGS", codesignopts)
```

`set_if_true` will only set on `build_settings` if `codesignopts` is non-empty. And it's an error to call this with a `None` `build_settings`.

```suggestion
    set_if_true(build_settings, "OTHER_CODE_SIGN_FLAGS", codesignopts)
```

---

Hmm, I guess we do check for non-`None` elsewhere. Do you can have that check still here.